src/components/ProfileUpdate/index.jsx

```diff
@@ -142,8 +142,14 @@ const ProfileUpdate = props => {
               <Select
                 value={gradYear}
                 className="year"
-                onBlur={value => props.setFieldValue('graduationYear', value)}
-                onChange={value => props.setFieldValue('graduationYear', value)}>
+                onBlur={value => {
+                  setGradYear(value);
+                  props.setFieldValue('graduationYear', value);
+                }}
+                onChange={value => {
+                  setGradYear(value);
+                  props.setFieldValue('graduationYear', value);
```

Comment:
This should be abstracted to the container with Formik, you can just follow the structure here in the container 
https://github.com/acmucsd/membership-portal-ui/blob/master/src/containers/ProfileUpdate.jsx. If you take a look at other input fields its these functions are passed down as prop as well. Typically we want to remove using anonymous functions within the jsx itself and pass these down as props or defined before the return. 

If you want to take a look at another example here is the container for RegisterForm https://github.com/acmucsd/membership-portal-ui/blob/master/src/containers/RegisterForm.jsx and the component here https://github.com/acmucsd/membership-portal-ui/blob/master/src/components/RegisterForm/index.jsx

Reply:
ReisterForm also uses anonymous functions in the JSX, I assume you'd want both forms to have the funcs moved?

Reply:
Yea same should be done with register but we can worry about profile update for now

Reply:
I've looked around, and Formik apparently cannot properly deal with Ant Design's selector. Turns out the idiomatic way to create custom Formik change handlers is to write them inline in the JSX, modifying props in the process.

I'll kick this can to the new review I requested.